omicsclaw/app/server.py

```
@@ -3460,6 +3508,89 @@ def provider_supports_oauth(_name: str) -> bool:  # type: ignore[no-redef]
+        api_key=runtime.api_key,
+        base_url=runtime.base_url or None,
+        timeout=15.0,
+    )
+    try:
+        response = await client.chat.completions.create(
+            model=runtime.model,
+            messages=[{"role": "user", "content": "Say OK"}],
+            max_tokens=8,
+        )
+        content = str(response.choices[0].message.content or "").strip()
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)